Source/monster.cpp

```diff
@@ -5027,7 +5022,7 @@ bool Monster::IsPossibleToHit() const
 	    || _mmode == MonsterMode::Charge)
 		return false;
 	if ((MType->mtype >= MT_COUNSLR && MType->mtype <= MT_ADVOCATE) && _mgoal != MGOAL_NORMAL) {
-		GenerateRnd(100); // Vanilla Compatibility Note: Pushing RNG, as original function CheckMonsterHit was called after CTH roll
+		// Vanilla Compatibility Note: put GenerateRnd(100); here, as original function CheckMonsterHit was called after CTH roll

```

Comment:
We don't retain notes like this for the other changes that we have made, having one here could be misleading.